index.html

```diff
@@ -250,38 +250,38 @@ <h2 class="header">About Myself</h2>
           </p>
           <a href="" class="button button-2">LET'S CONNECT</a>
           <ul class="profiles">
-            <li><a href="https://twitter.com/MAleemHashmi1"><img src="images/twitter.png"></a></li>
-            <li><a href="https://www.linkedin.com/in/muhammad-aleem-hashmi-09658a171/"><img src="images/linkedin.png"></a></li>
-            <li><a href="https://github.com/MAleemH"><img src="images/github.png"></a></li>
+            <li><a href="https://twitter.com/MAleemHashmi1"><img alt="not found" src="images/twitter.png"></a></li>
+            <li><a href="https://www.linkedin.com/in/muhammad-aleem-hashmi-09658a171/"><img alt="not found" src="images/linkedin.png"></a></li>
+            <li><a href="https://github.com/MAleemH"><img alt="not found" src="images/github.png"></a></li>
```

Comment:
- [ ] I would kindly suggest you give these social media images a real and descriptive `alternative text`, so it can be accessible for anyone coming to view the page but the image link got broken, and the `alternative text` will serve as a guide.

- I would suggest you give them an `alternative text` name that is peculiar to the social media name

For example 👇 
```HTML
<img alt="Twitter-image" src="images/twitter.png">
<img alt="LinkedIn-image" src="images/linkedin.png">
<img alt="Github-image" src="images/github.png">
```

- kindly change other places where you have your `alternative text` to be `not found` to be something peculiar to the image